python/ngen_cal/tests/conftest.py

```diff
@@ -151,12 +151,12 @@ def catchment(nexus, fabric, workdir, mocker) -> Generator[CalibrationCatchment,
     output = nexus._hydro_location.get_data().rename(columns={'value':'sim_flow'})
     output.set_index('value_time', inplace=True)
     #Override the output property so it doesn't try to reload output each time
-    mocker.patch(__name__+'.EvaluatableCatchment.output',
+    mocker.patch('ngen.cal.calibration_cathment.EvaluatableCatchment.output',
```

Comment:
Does the work to run tests without having an editable install?

Reply:
Works in both cases.

Reply:
Better question -- if a fresh clone is pulled, and you run the tests without doing any install, will they work?

Reply:
Required when using a `src` layout, it seems.  We should revisit this general topic, but it doesn't have to limit this PR if the package imports in other parts of the test code require an install to work.

Reply:
If possible, I suggest moving this to its own change set and/or issue.  It seems out of scope with the dead imports and style of the rest of this PR.

Reply:
I created #166 to track this. Putting it on my todos